<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mad Framebuffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QFrame" name="displayControlBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QComboBox" name="inSpaceCombo"/>
       </item>
       <item>
        <widget class="QPushButton" name="EvButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Ev</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="EvSlider">
         <property name="minimum">
          <number>-100</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="YButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="YSlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="outSpaceCombo"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GLWidget" name="BufferWindow" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Add_File_s"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuChannels">
    <property name="title">
     <string>channels</string>
    </property>
    <addaction name="actionIsolateR"/>
    <addaction name="actionIsolateG"/>
    <addaction name="actionIsolateB"/>
    <addaction name="actionIsolateA"/>
    <addaction name="actionIsolateRGBA"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionHistoryView"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChannels"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="historyDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="margin">
      <number>1</number>
     </property>
     <item>
      <widget class="QListWidget" name="historyList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string> &amp;Open File</string>
   </property>
  </action>
  <action name="action_Add_File_s">
   <property name="text">
    <string>&amp;Add File(s)</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionIsolateR">
   <property name="text">
    <string>isolateR</string>
   </property>
   <property name="shortcut">
    <string notr="true">R</string>
   </property>
  </action>
  <action name="actionIsolateG">
   <property name="text">
    <string>isolateG</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionIsolateB">
   <property name="text">
    <string>isolateB</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionIsolateA">
   <property name="text">
    <string>isolateA</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionIsolateRGBA">
   <property name="text">
    <string>isolateRGBA</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionHistoryView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>History View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <slot>outSpace_CB(int)</slot>
    <slot>inSpace_CB(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionHistoryView</sender>
   <signal>toggled(bool)</signal>
   <receiver>historyDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>historyDock</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionHistoryView</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outSpaceCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BufferWindow</receiver>
   <slot>outSpace_CB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inSpaceCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BufferWindow</receiver>
   <slot>inSpace_CB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHistoryView</sender>
   <signal>hovered()</signal>
   <receiver>historyDock</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>historyImageChanged(QListWidgetItem*,QListWidgetItem*)</slot>
 </slots>
</ui>
